conf/distro/nilrt.conf

```diff
@@ -2,9 +2,9 @@ require nilrt.inc
 
 DISTRO_NAME = "NI Linux Real-Time"
 
-DISTRO_VERSION = "8.7"
+DISTRO_VERSION = "8.8"
 
-NILRT_RELEASE_NAME = "2020.7"
+NILRT_RELEASE_NAME = "2021"
```

Comment:
This is probably going to be problematic. We use this variable to form feed URIs. Dunfell is already set to 2021..

Reply:
Yeah; I'm working on a fix for the dunfell feed version. I won't pull this until the dunfell feeds are moved.